package.json

```diff
@@ -23,6 +23,7 @@
     "watch:typecheck": "yarn build:typecheck -w",
     "start": "yarn workspaces foreach -vpi -j 1000 run start",
     "start:backend": "yarn workspace @asap-hub/asap-server start:server",
+    "start:frontend": "yarn workspace @asap-hub/frontend start",
```

Comment:
exactly what I have been running as well :sweat_smile: 
could also do storybook?

Reply:
I'm using TMUX to have all those tabs open. It's a bit too much :D
Should we just use: https://github.com/kimmobrunfeldt/concurrently ??

Reply:
Also using tmux. I would say everyone can decide that bit themselves, I think it's personal preference. I personally wouldn't use a script that spawn things in parallel because I want to have the output streams separately. Also, you always need different scripts active depending on what you're doing

Reply:
Yes, I do agree. I only see a use for it to run both babel and typechecking, but you can just use `&&` :D

Reply:
Srotybook added

Reply:
Yeah and mixing Babel output into typecheck would suck, typecheck needs to be readable